src/layers/wav2vec2.cc

```diff
@@ -1,57 +1,118 @@
 #include "ctranslate2/layers/wav2vec2.h"
+#include <iostream>
 
 
 namespace ctranslate2 {
   namespace layers {
+
+    Wav2Vec2LayerNormConvLayer0::Wav2Vec2LayerNormConvLayer0(const models::Model& model, const std::string& scope)
+      : _conv(model, scope + "/conv", /*stride=*/5, /*padding=*/0)
+      , _transpose({0, 2, 1})
+      , _output_norm(model, scope + "/layer_norm") {
+    }
+
+    void Wav2Vec2LayerNormConvLayer0::operator()(const StorageView& input, StorageView& output) const{
+      PROFILE("Wav2Vec2LayerNormConvLayer0");
+
+      StorageView buffer(input.dtype(), input.device());
+      buffer = std::move(input);
+      _conv(buffer, output);
+      _transpose(output, buffer);
+      _output_norm(buffer, output);
+      _transpose(output, buffer);
+      _gelu(buffer, output);
+    }
+
+    Wav2Vec2LayerNormConvLayer::Wav2Vec2LayerNormConvLayer(const models::Model& model, const std::string& scope)
+      : _conv(model, scope + "/conv", /*stride=*/2, /*padding=*/0)
+      , _transpose({0, 2, 1})
+      , _output_norm(model, scope + "/layer_norm") {
+    }
+
+    void Wav2Vec2LayerNormConvLayer::operator()(const StorageView& input, StorageView& output) const{
+      PROFILE("Wav2Vec2LayerNormConvLayer");
+
+      StorageView buffer(input.dtype(), input.device());
+      buffer = std::move(input);
+      _conv(buffer, output);
+      _transpose(output, buffer);
+      _output_norm(buffer, output);
+      _transpose(output, buffer);
+      _gelu(buffer, output);
+    }
```

Comment:
``Wav2Vec2LayerNormConvLayer0`` and ``Wav2Vec2LayerNormConvLayer0`` are quite similiar. Can you combine it into only one class? Just add a condition to build different _conv.